benchmarking/README.md

```diff
@@ -101,15 +101,22 @@ into a CSV file for analysis.
 For example (from the `FirstOrderLp.jl` root directory),
 
 ```sh
-$ julia --project=scripts scripts/solve_qp.jl \
---instance_path test/trivial_lp_model.mps --method pdhg \
---output_dir /tmp/first_order_lp_solve
-$ julia --project=scripts scripts/solve_lp_external.jl \
---instance_path test/trivial_lp_model.mps --solver scs-indirect \
---tolerance 1e-7 --output_dir /tmp/scs_solve
+$ benchmarking/collect_mittelmann_benchmark.sh /tmp ~/mittelmann_benchmark
+$ for INSTANCE in nug08-3rd qap15
+do
+  julia --project=scripts scripts/solve_qp.jl \
+    --instance_path ~/mittelmann_benchmark/$INSTANCE.mps.gz --method pdhg \
+    --output_dir /tmp/first_order_lp_solve_$INSTANCE
```

Comment:
The output dir doesn't need to vary with the instance name. The scripts allow results from multiple instance to be in the same directory. Instead, the directory should vary with the solver configuration, e.g., solve tolerance.

Reply:
Done.